CONTRIBUTING.md

```diff
@@ -87,3 +87,30 @@ You can enable it in VSCode with the following in your `settings.json`.
     }
   },
 ```
+
+# Testing
+
+We believe that testing that is also transparent is critically important to ensuring the integrity of Wormhole components. It is also critically important that everyone can independantly verify and extend Wormhole test cases, which allow us to further ensure that Wormhole components will operate as expected in both positive and especially negative test case scenarios
```

Comment:
This seems to say the same thing a little bit nicer perhaps? Also `independently` is the correct spelling.

```suggestion
We believe automated tests ensure the integrity of all Wormhole components. Anyone can verify or extend them transparently and they fit nicely with our software development lifecycle. This ensures Wormhole components operate as expected in both expected and failure cases.
```